src/components/Text/index.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import styled from 'styled-components';
+
+import UiText from '../UiText';
+
+const variants = Object.values(UiText.VARIANTS);
+
+const Text = styled(UiText).attrs(({ className }) => ({
+  className: cx('f-Text', className),
+}))`
+  color: var(--f-color-text--secondary);
+  text-align: ${({ isNumber }) => isNumber && 'right'};
+`;
+
+Text.displayName = 'Text';
```

Comment:
Should we add a babel plugin to automatically do that?

Reply:
👌 will do that